src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$cell-size: 100px;
+$coll-couny: 7;
```

Comment:
It seems like there is a typo in the variable name, it should be `$col-count` instead of `$coll-couny`